content/applications/websites/livechat/chatbots.rst

```diff
@@ -0,0 +1,214 @@
+========
+Chatbots
+========
+
+A *chatbot* is a program designed to mimic a conversation with a live human. Chatbots are assigned
+a script of pre-written steps to follow. These scripts are designed to anticipate a visitor's
+potential response, and lead them through a series of questions and answers the same way an live
+team member would. Chatbots can be customized to fill various roles, depending on the page of the
+website they are assigned and the messages included in the script, among other criteria.
+
+.. image:: chatbot/chatbot-visitor-view.png
+   :align: center
+   :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat.
+
+Build a chatbot
+===============
+
+Before creating a new chatbot, the :guilabel:`Live Chat` application must first be installed on the
```

Comment:
```suggestion
Before creating a new chatbot, the *Live Chat* application must first be installed on the
```

Did we land on `:guilabels:` or `italics` for app/module names? Genuinely can't keep track at this point but I'll make the suggestion here, and will try to keep my suggestions consistent

Reply:
I debated on this one because I believe we landed on italics, but since it involves a directive (installing the app), I went with guilabel. I am going to rewrite the sentence for clarity, but I think you are right, I think it is italics.